TurkishFilter/sections/specific.txt

```diff
@@ -422,7 +423,7 @@ vidmoxy.com,rapidvid.net##.rkbg
 ||yenimarmaragazetesi.com/dimg/banner/
 ||filmpaf.com/1/1.gif
 ||diziyoucdn.xyz/*.mp4
-/get-ad|$domain=bursahayat.com.tr|bursainternetgazetesi.com|bursasporum.com|bursaport.com|igfhaber.com|lifebursa.com|lodoshaber.com|sozbursa.com|yenidonem.com.tr|webursa.com|bandirmayasam.com.tr|bbursa.com|bursa5n1k.com|bursasehrengiz.com|dijitalhaber.com.tr|fotospor.com|gencfm.com.tr|haberindogrusu.com|kadinveekonomi.com|kaleninsesi.com|korkutelimanset.com|marmaratv.com.tr|otoritehaber.com|sokgazetesi.com.tr|bizimgazete.com.tr|bursaekspress.com|bilginhaberci.com
+/get-ad|$domain=sokgazetesi.com.tr|bursahakimiyet.com.tr|bursahayat.com.tr|bursainternetgazetesi.com|bursasporum.com|bursaport.com|igfhaber.com|lifebursa.com|lodoshaber.com|sozbursa.com|yenidonem.com.tr|webursa.com|bandirmayasam.com.tr|bbursa.com|bursa5n1k.com|bursasehrengiz.com|dijitalhaber.com.tr|fotospor.com|gencfm.com.tr|haberindogrusu.com|kadinveekonomi.com|kaleninsesi.com|korkutelimanset.com|marmaratv.com.tr|otoritehaber.com|sokgazetesi.com.tr|bizimgazete.com.tr|bursaekspress.com|bilginhaberci.com
```

Comment:
duplicate of `sokgazetesi.com.tr`

Reply:
@Alex-302 Which one is duplicate?

Reply:
`sokgazetesi.com.tr`
![image](https://user-images.githubusercontent.com/8361299/211110089-ed4a5a49-0c88-4bed-94f0-012927e0af14.png)